test/hitchhiker/tree/messaging_test.clj

```diff
@@ -153,9 +153,9 @@
   (mixed-op-seq 0.5 250 5000))
 
 (defspec test-many-keys-bigger-trees
-  1000
```

Comment:
If these aren't fast enough, then we could parameterize them based on the profile, so that we can ensure the Travis profile does thorough testing.